rust-toolchain

```diff
@@ -1 +1 @@
-1.69
+1.70
```

Comment:
I think this is encoded in Cargo.toml too.

Reply:
Thx